src/utils/headers.ts

```diff
@@ -5,22 +5,52 @@ import { updateCacheControlHeader } from './cache-control'
 import { CustomerVariables } from './customer-variables/customer-variables'
 import { getPreSharedSecret } from './customer-variables/selectors'
 
-const ALLOWED_RESPONSE_HEADERS = [
-  'access-control-allow-credentials',
-  'access-control-allow-origin',
-  'access-control-expose-headers',
-  'content-encoding',
-  'content-type',
-  'cross-origin-resource-policy',
-  'etag',
-  'vary',
-]
+const BLACKLISTED_HEADERS = new Set([
```

Comment:
Note that we also have to drop `Strict-Transport-Security`.

Reply:
+1